### PR TITLE
Add Repeat to the IFixture interface issue #540

### PR DESCRIPTION
**Pull request by SeanFarrow**
Reopening the pr as I inadvertently closed it!

### CODE REVIEW

File: Src/AutoFixtureUnitTest/FixtureTest.cs [hunk 1]
```diff
@@ -1847,12 +1847,11 @@ public void DefaultRepeatCountIsThree()
...
             // Teardown
         }
-
-        [Fact]
+            [Fact]
         public void RepeatWillPerformActionTheDefaultNumberOfTimes()
         {
             // Fixture setup
-            Fixture sut = new Fixture();
+            IFixture sut = new Fixture();
```
Comment (ploeh) [inline Src/AutoFixtureUnitTest/FixtureTest.cs]: Why was this test changed to work on `IFixture`? Doesn't it work on `Fixture` any longer?

**Comment by ploeh** [comment]
Looking good :+1: 

It looks like there are a few issues to address, but they look minimal to me.

**Comment by SeanFarrow** [comment]
I think there all fixed, I’ve reverted the tests to use the Fixture class and fixed all the documentation warnings.

Cheers

Sean.

**Comment by ploeh** [comment]
Almost there, but the `FixtureTest` code looks funny, now.

Basically, I'd expect that that file wasn't touched at all...

**Comment by ploeh** [comment]
Thank you for your contribution! It's now live as [AutoFixture 3.43.2](https://www.nuget.org/packages/AutoFixture/3.43.2).

**Pull request merged**